src/elements/common/messages.js

```diff
@@ -612,6 +612,11 @@ const messages = defineMessages({
         description: 'Message shown when there are no folder items.',
         defaultMessage: 'There are no items in this folder.',
     },
+    metadataState: {
```

Comment:
Is this message used? It looks like we're using `messages.loadingState` for `VIEW_METADATA`.

Reply:
Yes this is used. When it's not loading, to determine the empty state message,